system/build.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+# Make sure we've got the compiler
+if [ "`which arm-none-eabi-gcc`" == "" ]; then
+  if [ "`which apt`" != "" ]; then
+    echo "Installing gcc-arm-none-eabi..."
+    sudo apt install -y gcc-arm-none-eabi
+  else
+    echo "[ERROR] arm-none-eabi-gcc not found!"
+    exit 1
+  fi
+fi
+
+# Required device
+DEVICE="$1"
+if [ "$DEVICE" == "" ]; then
+  echo
+  echo "Usage: $0 [DEVICE] [THREADING]"
+  echo
+  echo " DEVICE"
+  echo "   ds203: Build for DS203"
+  echo "   ds213: Build for DS213"
+  echo "   la104: Build for LA104"
+  echo "   all:   Build for all the devices"
+  echo
+  echo " THREADING"
+  echo "   single: Build one app at a time (Default)"
+  echo "   multi:  Build all the apps in a dir at the same time"
+  echo "   all:    Build ALL the apps at the same time"
+  echo
+  exit 1
+fi
+
+# Should we go multi-threading?
+THREADED="$2"
+if [ "$THREADED" != "multi" -a "$THREADED" != "all" ]; then
+  THREADED="single"
+fi
+
+# If we have to build all of them
+if [ "$DEVICE" == "all" ]; then
+  for build in ds203 ds213 la104; do
+    echo "Building $build..."
+    $0 $build || exit 1
+  done
+fi
+
+if [ ! -f "../tools/elfstrip/elfstrip" ]; then
```

Comment:
Make sure elfstrip is built. Currently you have to manually build it, otherwise the `system/release/` output is not the expected one.